test/unit/utils.py

```diff
@@ -9,12 +9,11 @@ def input_path(self, filename):
 
   def roundtrip(self, filename, opts=[]):
     path = self.input_path(filename)
-    p = run_process(WASM_OPT + ['-g', '-o', '-', path] + opts, check=False,
-                    capture_output=True)
+    p = run_process(WASM_OPT + ['-g', '-o', 'a.wasm', path] + opts)
```

Comment:
Was the loss of `check=False` here intentional? I'm not sure that's a good change because unexpected exceptions are reported as errors instead of failures in the test runner.

Reply:
I'm not sure why one is better than the other? Both will fail with a clear-enough error I think, and it's less writing the way I left it. Is there another advantage to the old way I'm missing?

Reply:
It can be useful to maintain the distinction between ERRORS, which mean that the test failed to run completely, and FAILURES, which mean that the test ran but one of the test asserts failed. But you're right that it's a minor difference. I don't feel strongly about this.

Reply:
Fair enough. For the record, I sort of do feel strongly about this, since I think it's important to be able to write tests as simply "run this, run that, run this other thing, check this output", and to assume that if any of those runnings fail then the test fails (without writing a few lines of error handling for each one).